star-dash/star-dash/GameBridge/GameBridge.swift

```diff
@@ -6,6 +6,7 @@ class GameBridge {
     var scene: SDScene
 
     var entitiesMap: [EntityId: SDObject]
+    var objectMap: [SDObjectId: EntityId]
```

Comment:
maybe can rename to objectsMap? to be consistent with the above variable naming convention